tests/test_wave_03.py

```diff
@@ -54,13 +54,14 @@ def test_friends_unique_movies_not_duplicated():
 
     # Assert
     assert len(friends_unique_movies) == 3
-
-    raise Exception("Test needs to be completed.")
+    assert friends_unique_movies == [FANTASY_4, HORROR_1, INTRIGUE_3]
```

Comment:
The get_friends_unique_watched method doesn't specify the order of the movies. 

What if the list of `friends_unique_movies` was `[INTRIGUE_3, FANTASY_4, HORROR_1]` instead? Would this assertion still pass?

Since we don't specify or guarantee order of movies, we should prefer to use `in` to check only for each movie's presence in the list (without checking order).
```py
    assert FANTASY_4 in friends_unique_movies
    assert HORROR_1 in friends_unique_movies
    assert INTRIGUE_3 in friends_unique_movies